components/status/StatusActionsMore.vue

```diff
@@ -1,4 +1,6 @@
 <script setup lang="ts">
+import { env } from 'process'
+import { metaProperty } from '@babel/types'
```

Comment:
Are these imports needed?

Reply:
No, it was just ts trying to be too clever. I've already removed it.